src/components/app/index.tsx

```diff
@@ -95,7 +117,7 @@ export default class App extends Component<Props, State> {
     const { source, images } = this.state;
 
     for (const [i, image] of images.entries()) {
-      if (source !== prevState.source || image !== prevState.images[i]) {
+      if (source !== prevState.source || image.encoderState !== prevState.images[i].encoderState) {
```

Comment:
The image only needs updated if the encoder settings have changed (or the source has changed)

Reply:
Seems like your comment should be a code comment